core/src/main/resources/schema/app/20201130155647_CB-9580_new_field_on_cluster:_embedded_db_is_configured_on_attached_disk_or_not.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-9580 new field on cluster: embedded db is configured on attached disk or not
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE cluster ADD COLUMN IF NOT EXISTS embeddeddatabaseonattacheddisk boolean NOT NULL DEFAULT FALSE;
```

Comment:
This is the way to add a new column with a default value: https://dba.stackexchange.com/questions/60965/does-postgresql-optimize-adding-columns-with-non-null-defaults/216153#216153